.github/workflows/ci.yaml

```diff
@@ -213,6 +213,120 @@ jobs:
+          #   extratesting: go get -u github.com/sirupsen/logrus@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrawssdk-v1
+            extratesting: go get -u github.com/aws/aws-sdk-go@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrawssdk-v2
+            extratesting: go get -u github.com/aws/aws-sdk-go-v2@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrecho-v3
+            # Test against the latest v3 Echo:
+            extratesting: go get -u github.com/labstack/echo@v3
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrecho-v4
+            extratesting: go get -u github.com/labstack/echo/v4@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrelasticsearch-v7
+            extratesting: go get -u github.com/elastic/go-elasticsearch/v7@7.x
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrgin
+            extratesting: go get -u github.com/gin-gonic/gin@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrgorilla
+            extratesting: go get -u github.com/gorilla/mux@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrgraphgophers
+            extratesting: go get -u github.com/graph-gophers/graphql-go@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrlogrus
+            extratesting: go get -u github.com/sirupsen/logrus@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrlogxi
+            extratesting: go get -u github.com/mgutz/logxi@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrpkgerrors
+            extratesting: go get -u github.com/pkg/errors@master
+          # Skip v4 lambda tests until the lambda integration can be properly
+          # implemented considering the change to OpenTelemetry.
+          # - go-version: 1.14.x
+          #   dirs: v4/integrations/nrlambda
+          #   extratesting: go get -u github.com/aws/aws-lambda-go@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrmysql
+            extratesting: go get -u github.com/go-sql-driver/mysql@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrpq
+            extratesting: go get -u github.com/lib/pq@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrpq/example/sqlx
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrredis-v7
+            extratesting: go get -u github.com/go-redis/redis/v7@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrsqlite3
+            extratesting: go get -u github.com/mattn/go-sqlite3@master
+          - go-version: 1.14.x
+            dirs: v4/integrations/nrsnowflake
+            extratesting: go get -u github.com/snowflakedb/gosnowflake@master
+          # gRPC tests should be skipped until Distributed Tracing has been
+          # implemented.
```

Comment:
Could we activate all those tests now?

Reply:
Yup! I'd rather do it in a separate PR since it will take a bit of work.